client/signup/steps/import-url/index.jsx

```diff
@@ -60,13 +64,14 @@ class ImportURLStepComponent extends Component {
 			} );
 		}
 
-		this.goToNextStep( {} );
+		this.props.fetchIsSiteImportable( this.props.urlInputValue ).then( () => this.goToNextStep() );
```

Comment:
`.then( this.goToNextStep )` should suffice right?

Reply:
done.  thanks